pkg/tidbver/tidbver.go

```diff
@@ -67,6 +67,12 @@ func TiFlashNotNeedSomeConfig(version string) bool {
 	return semver.Compare(version, "v5.4.0") >= 0 || strings.Contains(version, "nightly")
 }
 
+// TiFlashSupportRunByArg return true if the given version of TiFlash could be started without
+// a config file.
+func TiFlashSupportRunWithoutConfig(version string) bool {
```

Comment:
Excellent work! Do you plan to add this function to to tiup-cluster?

Reply:
Currently I don't have plans, because there seems to be no functional difference to users. If there are benefits and I think its worth to do!